src/Java.Interop.Tools.Maven/Repositories/InMemoryMavenRepository.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Threading.Tasks;
+using Java.Interop.Tools.Maven.Models;
+
+namespace Java.Interop.Tools.Maven.Repositories;
+
+public class InMemoryMavenRepository : IMavenRepository
+{
+	readonly Dictionary<string, byte []> cache = new ();
+
+	public string Name => "InMemory";
+
+	public async Task AddFile (Artifact artifact, string filename, Stream stream)
```

Comment:
Should this follow convention, and have an `Async` suffix?

Reply:
Even better, this type was used in an earlier design but is no longer used, so I removed it.